### ISSUE TITLE
Paxel plugin causes all tools to have negative attack speed

### ISSUE THREAD

**Issue by suuift** [PoorgrammerDev/VanillaPlus#2]
Paxel plugin causes all tools including vanilla to have negative attack speed and everything that that affects:

- causes weapons to only do minimum attack damage
- tools no longer show up in player hand

**Comment by PoorgrammerDev** [comment]
Hey there, I just re-read your issue. Just to make sure, you said this is affecting **all** tools, not just the Paxel?

In my testing I noticed that the attack cooldown is depleted on the Paxel itself when it has to switch between its three internal tools (i.e. when you begin digging a block of a different type than before). This should not affect the Paxel all the time, and it definitely shouldn't affect other tools in any capacity.

What Spigot/Minecraft version are you using this plugin with, and are there any other plugins present? Thanks!

**Comment by suuift** [comment]
It affected all tools in the world, even in creative. I found out somehow it had set my player attack speed to a negative value. Once I set it back to default I haven't had any issues.

Using purpur-1.20.1-2023

Using plugins: Citizens, dynmap, KeepChunks, mcMMO, Paxel, RHLeafDecay, Sentinel, SilkSpawners_v2, spark, UltimateHomes, UltimateInventory, VeinMiner, WildLoaders


I also just wanted to say great plugin btw! The speed it switches and interaction with enchantments and everything is great

**Comment by PoorgrammerDev** [comment]
Thank you suuift, I'm glad you're enjoying the plugin!

As for the error, I combed through the code and couldn't find anything that changes the player's attack speed attribute to a negative value.
I also downloaded the Purpur 1.20.1 server and tested the plugin on there, but I couldn't replicate the issue.

I'll continue looking into this in my spare time, but I'm inclined to say that this was caused by a different plugin.
In any case, thank you for your report, and I'm glad to hear it was easily fixable in your case!
